test/test_common/utility.cc

```diff
@@ -181,11 +181,13 @@ void TestUtility::waitForGaugeEq(Stats::Store& store, const std::string& name, u
   }
 }
 
-std::list<Network::Address::InstanceConstSharedPtr>
-TestUtility::makeDnsResponse(const std::list<std::string>& addresses) {
-  std::list<Network::Address::InstanceConstSharedPtr> ret;
+std::list<Network::DnsResponse>
+TestUtility::makeDnsResponse(const std::list<std::string>& addresses,
+                             const std::chrono::seconds ttl) {
```

Comment:
nit: del const

Reply:
I don't think we can delete `const` and keep `&` at the same time. There are some places calling this function by passing right value, for example, https://github.com/envoyproxy/envoy/blob/4b76a2f456e0771faa8d1e444df8e3a8f067c73b/test/common/upstream/upstream_impl_test.cc#L307

Reply:
I meant to remove the `const` on the `ttl` param. It doesn't do anything.